### PR TITLE
Tweaks on people tab

### PR DESCRIPTION
**Pull request by cecileboucheron**
@keybase/react-hackers

### CODE REVIEW

**Comment by buoyad** [comment]
The follow icons seem a little high on mobile, is that intentional?
779fa831-d3a3-4fd0-880d-48979aa7bd9b

**Comment by cecileboucheron** [comment]
Yeah I changed it! It's intentional.

File: shared/people/item/index.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ export default (props: Props) => (
       {props.badged && (
         <Badge
           badgeNumber={null}
-          badgeStyle={{marginLeft: globalMargins.tiny, height: 10, width: 10, minWidth: 10}}
+          badgeStyle={{marginLeft: globalMargins.tiny, height: 8, width: 8, minWidth: 10}}
```
Comment (buoyad) [inline shared/people/item/index.js]: `minWidth: 10 -> 8`

Comment (cecileboucheron) [reply shared/people/item/index.js]: Yeah it was actually 8, not 10.

Comment (buoyad) [reply shared/people/item/index.js]: Yeah, I mean `minWidth` should match `width` here or else the desktop badge looks a little stretched:
image

Comment (cecileboucheron) [reply shared/people/item/index.js]: Ha! Ok updated.

**Pull request merged**